experiment/run_experiment.py

```diff
@@ -59,36 +59,54 @@
 _OSS_FUZZ_CORPUS_BACKUP_URL_FORMAT = (
     'gs://{project}-backup.clusterfuzz-external.appspot.com/corpus/'
     'libFuzzer/{fuzz_target}/public.zip')
+DEFAULT_MAX_CONCURRENT_BUILDS = 30
 
 
+def set_default_config_values(config: Dict):
+    """Set the default configuration values if they are not specified."""
+    config['concurrent_builds'] = config.get('concurrent_builds',
+                                             DEFAULT_MAX_CONCURRENT_BUILDS)
+
+
+# pylint: disable=too-many-locals, too-many-statements, too-many-branches
 def read_and_validate_experiment_config(config_filename: str) -> Dict:
     """Reads |config_filename|, validates it, finds as many errors as possible,
     and returns it."""
     config = yaml_utils.read(config_filename)
+    set_default_config_values(config)
     filestore_params = {'experiment_filestore', 'report_filestore'}
     cloud_config = {'cloud_compute_zone', 'cloud_project'}
     docker_config = {'docker_registry'}
-    string_params = cloud_config.union(filestore_params).union(docker_config)
-    int_params = {'trials', 'max_total_time'}
+    worker_config = {'worker_pool_name'}
+    string_params = cloud_config.union(filestore_params).union(
+        docker_config).union(worker_config)
+    int_params = {'trials', 'max_total_time', 'concurrent_builds'}
```

Comment:
Can you change the name of concurrent_builds to concurrent_cloud_builds since it's not repected for local builds.

Reply:
Done, but I am a bit confused here.
Do we still have a parameter to set [concurrent build value for local builds](https://github.com/google/fuzzbench/blob/da05da0dcb3680deee9d894d1bfa61edbbf4a90c/docs/running-a-local-experiment/running_a_local_experiment.md?plain=1#L95) or shall I remove that line from our doc?
Allowing concurrent local builds sounds like a useful feature to me, maybe we could use that value for both local and cloud builds?

Reply:
I think I was confused sorry, I thought it wasn't respected for local. Ignore this suggestion please. This is definitely a useful feature for local.